src/Propel/Generator/Model/Database.php

```diff
@@ -574,7 +574,7 @@ public function removeSequence($sequence)
     {
         if ($this->sequences) {
             if (false !== ($idx = array_search($sequence, $this->sequences))) {
-                unset($this->sequence[$idx]);
+                unset($this->sequences[$idx]);
```

Comment:
This is an interesting bug that PHPStan detected
The wrong property was unset here.